base/src/com/google/idea/blaze/base/command/info/BlazeInfoRunnerImpl.java

```diff
@@ -39,23 +38,23 @@ public ListenableFuture<byte[]> runBlazeInfoGetBytes(
       BuildInvoker invoker,
       BlazeContext context,
       List<String> blazeFlags,
-      String key) {
+      String... keys) {
     return BlazeExecutor.getInstance()
-        .submit(
-            () -> {
-              BlazeCommand.Builder builder = BlazeCommand.builder(invoker, BlazeCommandName.INFO);
-              builder.addBlazeFlags(blazeFlags);
-              if (key != null) {
-                builder.addBlazeFlags(key);
-              }
-              try (BuildResultHelper buildResultHelper = invoker.createBuildResultHelper();
-                  InputStream blazeInfoStream =
-                      invoker
-                          .getCommandRunner()
-                          .runBlazeInfo(project, builder, buildResultHelper, context)) {
-                return blazeInfoStream.readAllBytes();
-              }
-            });
+               .submit(
```

Comment:
where does this formatting change come from?

Reply:
I can't figure out how to set up the ide formatter to match what the project has in all details. Some things slip.

Reply:
do you have https://github.com/google/google-java-format ?

Reply:
That's where it was :) Let me use it.